builder/alicloud/ecs/image_config.go

```diff
@@ -9,37 +11,187 @@ import (
+	//     default, if this is an extra data disk, Packer will not encrypt the
+	//     data disk. Otherwise, Packer will keep the encryption setting to what
+	//     it was in the source image. Please refer to Introduction of [ECS disk encryption](https://www.alibabacloud.com/help/doc-detail/59643.htm)
+	//     for more details.
+	//
+	ECSImagesDiskMappings []AlicloudDiskDevice `mapstructure:"image_disk_mappings" required:"false"`
 }
 
 type AlicloudImageConfig struct {
-	AlicloudImageName                 string            `mapstructure:"image_name"`
-	AlicloudImageVersion              string            `mapstructure:"image_version"`
-	AlicloudImageDescription          string            `mapstructure:"image_description"`
-	AlicloudImageShareAccounts        []string          `mapstructure:"image_share_account"`
-	AlicloudImageUNShareAccounts      []string          `mapstructure:"image_unshare_account"`
-	AlicloudImageDestinationRegions   []string          `mapstructure:"image_copy_regions"`
-	AlicloudImageDestinationNames     []string          `mapstructure:"image_copy_names"`
-	ImageEncrypted                    *bool             `mapstructure:"image_encrypted"`
-	AlicloudImageForceDelete          bool              `mapstructure:"image_force_delete"`
-	AlicloudImageForceDeleteSnapshots bool              `mapstructure:"image_force_delete_snapshots"`
-	AlicloudImageForceDeleteInstances bool              `mapstructure:"image_force_delete_instances"`
-	AlicloudImageIgnoreDataDisks      bool              `mapstructure:"image_ignore_data_disks"`
-	AlicloudImageSkipRegionValidation bool              `mapstructure:"skip_region_validation"`
-	AlicloudImageTags                 map[string]string `mapstructure:"tags"`
-	AlicloudDiskDevices               `mapstructure:",squash"`
+	// The name of the user-defined image, [2, 128]
+	// English or Chinese characters. It must begin with an uppercase/lowercase
+	// letter or a Chinese character, and may contain numbers, _ or -. It
+	// cannot begin with http:// or https://.
+	AlicloudImageName string `mapstructure:"image_name" required:"true"`
+	// The version number of the image, with a length
+	// limit of 1 to 40 English characters.
+	AlicloudImageVersion string `mapstructure:"image_version" required:"false"`
+	// The description of the image, with a length
+	// limit of 0 to 256 characters. Leaving it blank means null, which is the
+	// default value. It cannot begin with http:// or https://.
+	AlicloudImageDescription string `mapstructure:"image_description" required:"false"`
+	// The IDs of to-be-added Aliyun
+	// accounts to which the image is shared. The number of accounts is 1 to 10.
+	// If number of accounts is greater than 10, this parameter is ignored.
+	AlicloudImageShareAccounts   []string `mapstructure:"image_share_account" required:"false"`
+	AlicloudImageUNShareAccounts []string `mapstructure:"image_unshare_account"`
```

Comment:
So, when a field is not documented the text from the field name is split & used there; ex: https://deploy-preview-7724--packer-www.netlify.com/docs/builders/alicloud-ecs.html#image_unshare_account looks like this:
<img width="515" alt="Screenshot 2019-08-27 at 09 06 51" src="https://user-images.githubusercontent.com/1024852/63748667-11caa700-c8aa-11e9-8329-40c7cc29ae0b.png">

I did this because I didn't really want to search for these missing informations & may be we can fix this in later issues.

Reply:
Do you think I should do something else about it/them ?

Reply:
That seems fair.